### PR TITLE
Add Dark Castle Color Scheme

### PR DESCRIPTION
**Pull request by scottgriv**
- [X] I'm the package's author and/or maintainer.
- [X] I have have read [the docs][1].
- [X] I have tagged a release with a [semver][2] version number.
- [X] My package repo has a description and a README describing what it's for and how to use it.
- [X] My package doesn't add context menu entries. *
- [X] My package doesn't add key bindings. **

My package is similar to color schemes in general... However it should still be added because ...
I'm creating a new multi-platform theme, that's unlike any theme I've personally come across.
I've built it up for a [number of platforms](https://darkcastle.netlify.app/), and I would love to have Sublime included in my list.

Thanks!

### CODE REVIEW

File: repository/d.json [hunk 1]
```diff
@@ -113,6 +113,18 @@
 				}
 			]
 		},
+		{
+			"name": "Dark Castle Color Scheme",
+			"author": "Scott Grivner",
+			"details": "https://github.com/scottgriv/Dark-Castle-Sublime",
+			"labels": ["theme", "color scheme", "dark theme", "dark castle", "dark"],
```
Comment (braver) [inline repository/d.json:120]: In Sublime Text parlance this is a color scheme and _not_ a theme, so it should also not have that label. Repeating your theme’s name as a label also feels a bit superfluous.

**Review by braver — CHANGES_REQUESTED** [review]
Also have a look at the bot’s feedback please!

File: repository/d.json (hunk 1 above)
Comment (scottgriv) [reply repository/d.json:120]: Good points. Thank you for your feedback. I'll make the appropriate changes.

**Comment by braver** [comment]
Package Control only looks at your tags, so if you made an improvement please also tag a new release.

**Comment by scottgriv** [comment]
Thanks for your help.

**Pull request merged**